From: Sven Eckelmann <sven@narfation.org>
Date: Wed, 25 Apr 2018 21:18:35 +0200
Subject: batman-adv: upgrade package to latest release 2018.1

* support latest kernels (3.2 - 4.17)
* coding style cleanups and refactoring
* add DAT cache and multicast flags netlink support
* avoid redundant multicast TT entries
* bugs squashed:

  - update data pointers after skb_cow()
  - fix header size check in batadv_dbg_arp()
  - fix skbuff rcsum on packet reroute
  - fix multicast-via-unicast transmission with AP isolation
  - fix packet loss for broadcasted DHCP packets to a server
  - fix multicast packet loss with a single WANT_ALL_IPV4/6 flag

Signed-off-by: Sven Eckelmann <sven@narfation.org>

diff --git a/batman-adv/Makefile b/batman-adv/Makefile
index 3fd463bdeefc1b263dfe636953bf59531785a66f..16b4ae6d3dcd7553036ec1b1a306d74d1f00c030 100644
--- a/batman-adv/Makefile
+++ b/batman-adv/Makefile
@@ -9,10 +9,9 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=batman-adv
 
-PKG_VERSION:=2018.0
+PKG_VERSION:=2018.1
 PKG_RELEASE:=0
-PKG_MD5SUM:=c39d67cdb5509bf638efc9083e0dd41e
-PKG_HASH:=4826f838e8a2914a9470da25ea2f17f6325c464a130093b20dc7fb4e93f7576c
+PKG_HASH:=b866b28dbbe5c9238abbdf5abbc30fc526dea56898ce4c1bd76d5c017843048b
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://downloads.open-mesh.org/batman/releases/batman-adv-$(PKG_VERSION)
